parfait-core/src/test/java/io/pcp/parfait/timing/EventMetricCollectorTest.java

```diff
@@ -60,7 +63,7 @@ public void givenAnEventMetricCollector() {
         when(nestedCounter.getEventGroupName()).thenReturn(NESTED_GROUP);
         EventMetricCounters invocationCounter = mock(EventMetricCounters.class);
         when(topLevelCounter.getInvocationCounter()).thenReturn(invocationCounter);
-        ThreadMetric threadMetric = mock(ThreadMetric.class);
+        ThreadMetric threadMetric = new DummyThreadMetric(AbstractUnit.ONE);
```

Comment:
I tried making this work with Mokito and gave up so I'll just leave this here for now. :-D

Reply:
Ok thanks, I assume somehow playing with MockSettings it might be doable, but I also don't have more time and the Dummy class is only in the test code, so it looks legitimate for testing purposes.
Who can merge such PRs?